packages/wdio-allure-reporter/src/index.js

```diff
@@ -120,6 +124,20 @@ class AllureReporter extends WDIOReporter {
         currentTest.addLabel('thread', cid)
     }
 
+    getLabels(test){
+        var tags = test.tags
+        var labels = []
+        if (tags != null && tags != undefined){
```

Comment:
`tags != null && tags != undefined` could be replaced with `!!tags`

Reply:
Done